src/main/routes.ts

```diff
@@ -155,6 +159,71 @@ export class Routes {
     app.get(C100_RETRIVE_CASE, errorHandler(new GetCaseController().getC100ApplicantCase));
     app.get(C100_DOWNLOAD_APPLICATION, errorHandler(new ApplicationDownloadController().download));
 
+    //Tasklist event common get controller routes
+    app.get(
+      `${RESPONDENT_DETAILS_KNOWN}/:caseId`,
+      errorHandler(new TasklistresponseCommonGetController(EventRoutesContext.KEEP_DETAILS_PRIVATE_RESPONDENT).get)
```

Comment:
Change the name to TasklistGetController

Reply:
Done.